net/http/src/THttpEngine.cxx

```diff
@@ -36,5 +36,5 @@ THttpEngine::THttpEngine(const char *name, const char *title)
 
 THttpEngine::~THttpEngine()
 {
-   fServer = 0;
+   fServer = nullptr;
```

Comment:
You can safely remove this line - after `~THttpEngine()`, any access to its `this->fServer` would be a disaster anyway.

Reply:
> would be a disaster anyway

You are right. 

From other side, correct cleanup of objects in multi-threaded environment is extremely complex. And such dirty tricks allows to code survive in case of wrong cleanup sequences. 

I will remove it.